includes/Hooks.php

```diff
@@ -473,4 +508,151 @@ public function onLogEventsListLineEnding(
 		// Add parentheses to match what's done with Thanks in revision lists and diff displays.
 		$ret .= ' ' . wfMessage( 'parentheses' )->rawParams( $thankLink )->escaped();
 	}
+
+	/**
+	 * Fandom change UGC-4012 - Add thank link to the recent changes list
+	 *
+	 * @link https://www.mediawiki.org/wiki/Manual:EnhancedChangesListModifyLineDataHook.php
+	 * @param EnhancedChangesList $changesList
+	 * @param array &$data
+	 * @param RecentChange[] $block
+	 * @param RecentChange $rc
+	 * @param string[] &$classes
+	 * @param string[] &$attribs
+	 * @return void
+	 */
+	public function onEnhancedChangesListModifyLineData( $changesList, &$data, $block, $rc, &$classes, &$attribs ) {
```

Comment:
Suggestion: Some of these handlers could be moved to `FandomThanks/Handler/MobileDiffHandler` this way we could reduce our fork size.
Also we could move `ThanksCache` and `ThanksPermissions` classes to `FandomThanks`